### PR TITLE
ref: Remove identity-wallet dep, use did:key

### PR DESCRIPTION
**Pull request by oed**
This PR removes `identity-wallet` as a dependency and replaces it with `key-did-provider-ed25519`.

### CODE REVIEW

File: packages/cli/src/ceramic-cli-utils.ts [hunk 1]
```diff
@@ -329,9 +329,8 @@ export class CeramicCliUtils {
         }
 
         const seed = u8a.fromString(cliConfig.seed)
```
Comment (simonovic86) [inline packages/cli/src/ceramic-cli-utils.ts:331]: I guess we need to update the `seed` generation as well, right?

```js
if (!cliConfig.seed) {
  cliConfig.seed = u8a.toString(randomBytes(32))
  console.log('Identity wallet seed generated')
  await CeramicCliUtils._saveCliConfig(cliConfig)
}
```

**Review by simonovic86 — COMMENTED** [review]
nice! I left a comment related to the CLI and non-existing `seed`

**Review by PaulLeCam — APPROVED** [review]

File: packages/cli/src/ceramic-cli-utils.ts (hunk 1 above)
Comment (oed) [reply packages/cli/src/ceramic-cli-utils.ts:331]: if it's 32 bytes it should be correct @simonovic86

Comment (oed) [reply packages/cli/src/ceramic-cli-utils.ts:331]: Everywhere in the tests we had 64 bytes for some reason 🤷

Comment (simonovic86) [reply packages/cli/src/ceramic-cli-utils.ts:331]: nice! 🙂

**Review by simonovic86 — APPROVED** [review]

**Pull request merged**